tss-esapi/src/tcti_ldr.rs

```diff
@@ -204,6 +210,9 @@ impl TryFrom<TctiNameConf> for CString {
             TctiNameConf::Tabrmd(config) => {
                 format!("bus_name={},bus_type={}", config.bus_name, config.bus_type)
             }
+            TctiNameConf::LibTpms { state } => state
+                .and_then(|s| s.to_str().map(str::to_string))
```

Comment:
Question: If a path have been provided would it make any sense to check if it is valid?

Reply:
I would argue this is the responsibility of the TCTI backend?